course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
+1. Why is it beneficial to use a programming language rather than a spreadsheet software for data analyses?
+2. Get setup with Rstudio at home (or a lab computer after hours). Install R, Rstudio, and the necessary packages (tidyverse, rmarkdown). Use `install.packages(<package_name>)`. Load all the libraries you installed into your environment with `library(<package_name>)` and send us the output of running `sessionInfo()`.
```

Comment:
It would be nice, however... that would require installing MikTeX or whatever equivalent is available. This may or may not be too much work (installing LaTeX can sometimes be a major pain). Are we able to get LaTeX installed on the computers beforehand? We'll have to mention that if they do this on their own laptops or whatever, that it might not work/take a lot of effort to install.